modules/rosa-upgrading-preparing-4-7-to-4-8.adoc

```diff
@@ -24,7 +24,7 @@ You must meet the following requirements before upgrading a {product-title} (ROS
 +
 [source,terminal]
 ----
-$ rosa create account-roles --mode auto --version 4.8
```

Comment:
we probably don't need to remove the `--mode auto` ?  most people will likely want to do auto mode.